nbs/20_comparison_others/py/03_02-gtex_whole_blood-sample_all_gene_pairs.py

```diff
@@ -0,0 +1,121 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: all,-execution,-papermill,-trusted
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.5
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown] tags=[]
+# # Description
+
+# %% [markdown] tags=[]
+# It samples all the universe of gene pairs within the top genes initially selected (5,000 genes with maximum variance). This sample is needed to compare Clustermatch with other coefficients that are slower, like Maximal Information Coefficient.
+
+# %% [markdown] tags=[]
+# # Modules
+
+# %% tags=[]
+import pandas as pd
+
+from clustermatch import conf
+
+# %% [markdown] tags=[]
+# # Settings
+
+# %% tags=[]
+DATASET_CONFIG = conf.GTEX
+GTEX_TISSUE = "whole_blood"
+GENE_SEL_STRATEGY = "var_pc_log2"
+
+# %% tags=[]
+# amount of gene pairs to sample
+SAMPLE_SIZE = 100000
```

Comment:
Are you setting the random seed anywhere?

Reply:
Great question. Yes, when I run the `sample` method from each dataframe, I set the `random_state` argument to the sample id value.